lib/std/fs.zig

```diff
@@ -507,6 +507,7 @@ pub const SelfExePathError = error{
     DeviceBusy,
     SharingViolation,
     PipeBusy,
+    PipeNotAvailable,
```

Comment:
I wonder if we'd prefer `PipeUnavailable`? Not sure if we have any real naming convention for stuff like this.